CMakeLists.txt

```diff
@@ -37,6 +37,15 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_C_EXTENSIONS OFF)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Set manifest mode to use vcpkg.json to resolve dependencies
+option(VCPKG_MANIFEST_MODE ON)
+
+# When ON, vcpkg install is ran every time configure is ran
+option(VCPKG_MANIFEST_INSTALL ON)
```

Comment:
Are you sure that this works? The CMake documentation seems to indicate that the help text is required (see https://cmake.org/cmake/help/latest/command/option.html), so I wonder if in this case it is using "ON" as help text and assuming the default value for the option of `OFF`.

Reply:
I'll add an help text just in case.